frontend/src/utils/index.js

```diff
@@ -406,12 +399,31 @@ export function isTypeDelete (lastOperation) {
   return get(lastOperation, 'type') === 'Delete'
 }
 
-export function isServiceAccount (username) {
+export function isServiceAccountUsername (username) {
   return startsWith(username, 'system:serviceaccount:')
 }
 
-export function isServiceAccountFromNamespace (username, namespace) {
-  return startsWith(username, `system:serviceaccount:${namespace}:`)
+export function isForeignServiceAccount (currentNamespace, serviceAccountName) {
+  if (serviceAccountName && currentNamespace) {
+    const { namespace } = nameAndNamespaceFromServiceAccountUsername(serviceAccountName)
+    if (namespace && namespace !== currentNamespace) {
+      return true
+    }
+  }
+  return false
+}
+export function nameFromServiceAccountUsername (serviceAccount) {
+  if (serviceAccount) {
+    return last(split(serviceAccount, ':'))
+  }
+}
+
+export function nameAndNamespaceFromServiceAccountUsername (username) {
```

Comment:
rename `nameAndNamespaceFromServiceAccountUsername` to `parseServiceAccountUsername`